### PR TITLE
Add star redirect

### PR DESCRIPTION
**Pull request by bitsondatadev**

### CODE REVIEW

File: netlify.toml [hunk 1]
```diff
@@ -17,3 +17,7 @@
   from = "/docs/*"
   to = "https://trinodb.github.io/docs.trino.io/:splat"
   status = 200
+
+[[redirects]]
+  from = "/star"
+  to = "https://github.com/trinodb/trino/blob/master/.github/star.png"
```
Comment (mosabua) [inline netlify.toml:23]: Add empty line at end of file

**Review by mosabua — APPROVED** [review]
Personally I would thinks its better to use an actual JS or HTML redirect in a file rather since that works whereever (including local testing and such) rather than a config specific to our hosting, but this will work as well. Up to the maintainers to approve and merge or change course.

File: netlify.toml (hunk 1 above)
Comment (bitsondatadev) [reply netlify.toml]: done

Comment (bitsondatadev) [reply netlify.toml]: It doesn't show up in the diff or online editor, but when I go to my branch and edit via online editor it shows the newline at the end.

**Comment by bitsondatadev** [comment]
> Personally I would thinks its better to use an actual JS or HTML redirect in a file since that works where ever (including local testing and such) rather than a config specific to our hosting, but this will work as well. Up to the maintainers to approve and merge or change course.

I'm good with updating this to html/js redirect. @martint or @electrum?

**Comment by mosabua** [comment]
Replaced by https://github.com/trinodb/trino.io/pull/188

**Comment by electrum** [comment]
HTTP redirect seems like the right way. We can consider if we want HTML redirect for local usage, but I feel like that is confusing and unnecessary. You can test that the redirect works using the Netlify preview for PRs.

**Pull request merged**